### PR TITLE
add uptick add coinImage

### PR DESCRIPTION
**Pull request by xianxianzhou**
update uptick_117.json add coinImage url

### CODE REVIEW

**Comment by HeesungB** [comment]
@xianxianzhou 
Is `Uptick` registered on `Coingecko`?
I couldn't find it, so any links would be appreciated.

If not, please delete `coingecko id` field.

**Comment by xianxianzhou** [comment]
CoingeckoiD has deleted

**Comment by HeesungB** [comment]
@xianxianzhou Hello.
Could you remove `walletUrl` and `walletUrlForStaking` field?
Because It is invalid.

**Comment by xianxianzhou** [comment]
I have deleted walletUrl,I would like to know if keplr can see the  logo of the currency after the current node information is registered.

File: cosmos/uptick_117.json [hunk 1]
```diff
@@ -28,16 +29,18 @@
...
 		"gasPriceStep": {
-			"low": 10000000000,
-			"average": 20000000000,
-			"high": 40000000000
+			"low": 125000000000,
+			"average": 130000000000,
+			"high": 150000000000
 		}
 	}],
 	"coinType": 60,
```
Comment (HeesungB) [inline cosmos/uptick_117.json:46]: @xianxianzhou Could you remove `coinType`field?

Comment (xianxianzhou) [reply cosmos/uptick_117.json]: I have remove coinType now

**Review by bitcoinfinancier — APPROVED** [review]

**Comment by HeesungB** [comment]
@xianxianzhou `Node Provider` is missing.
Could you add `Node Provider`?

**Comment by HeesungB** [comment]
@xianxianzhou, @bitcoinfinancier Thank you for contribution!

**Pull request merged**